src/blog/2024/06/dashboard-multi-tenancy.md

```diff
@@ -21,7 +21,7 @@ Having taken that feedback on board, we've made some significant changes to how
 A quick summary of the changes are as follows:
 
 - **Socket constraints moved to core Dashboard:** The ability to constrain communications to a specific client (using the `msg._client.socketId`) has been moved to the core of Dashboard 2.0, and is no longer a feature of the FlowFuse User Addon. As such, it's available to _all_ users, for _all_ node types.
-- **FlowFuse User Addon now Source Available:** The addon is available to install in _any_ instances of Node-RED running on FlowFuse, so that's all team tiers on FlowFuse Cloud, and any self-hosted instances of FlowFuse too. It is available to install from Node-RED's Palette Manager.
+- **FlowFuse User Addon now Source Available:** The addon is available to install in _any_ instances of Node-RED running on FlowFuse, so that's all pro tiers on FlowFuse Cloud, and any self-hosted instances of FlowFuse too. It is available to install from Node-RED's Palette Manager.
```

Comment:
```suggestion
- **FlowFuse User Addon now Source Available:** The addon is available to install in _any_ instances of Node-RED running on FlowFuse, so that's all team tiers on FlowFuse Cloud, and any self-hosted instances of FlowFuse too. It is available to install from Node-RED's Palette Manager.
```
Incorrectly changed